### PR TITLE
Update open-source-repository-migration.md

### PR DESCRIPTION
**Pull request by pauladamgordon**

### CODE REVIEW

**Review by Montana — COMMENTED** [review]
Looks good, not clear on the anchoring problem needs more detail.

**Review by Montana — APPROVED** [review]

**Review by mustafa-travisci — APPROVED** [review]
Looks good to me as well!

**Review by AlonaHlobina — APPROVED** [review]
Thank you for doing this, Paul! 
It looks good to me.

**Review by r-arek — APPROVED** [review]

File: user/migrate/open-source-repository-migration.md [hunk 1]
```diff
@@ -185,3 +185,49 @@ If you have any further questions, comments or need help on our Beta migration p
...
+
+
+#### Q. Will there be lower concurrency for free accounts on travis-ci.org?
+
+A. As part of the shift of infrastructure from .org to .com and needing to make sure all users have equal access to resources, free and open-source .org accounts will have concurrency reduced from 5 to 4 concurrent jobs. Concurrent jobs have not changed on .com, so please consider migrating your repositories as soon as possible if this is an issue.
+
+
+#### Q. Will Travis-ci.org become unreliable?
+
+A. As part of the shift of infrastructure from .org to .com we’re making a number of changes to the travis-ci.org infrastructure to ensure the service will remain as reliable and available to you as it always has been until the
```
Comment (hugovk) [inline user/migrate/open-source-repository-migration.md:218]: This sentence is incomplete.

**Pull request merged**